CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - [Web] Add views for handlers
 
 ### Changed
-- [Web] Updated embedded web assets from `9d91d7f` ... `275386a`
+- [Web] Vendored `sensu/web` @ SHA `0e6c347`
```

Comment:
Honestly, I think this line is redundant. We only store the embedded assets. So that's all I think we really care about.